examples/analytics-vendors.amp.html

```diff
@@ -55,6 +55,7 @@
       <option>baiduanalytics</option>
       <option>blackcrowai</option>
       <option>blueconic</option>
+      <option>bluetriangle</option>
```

Comment:
Was this not added in the previous PR that added Blue Triangle vendor?

Reply:
Hi, I was doing a double check and it looks like it wasn't